tensorflow/go/README.md

```diff
@@ -118,6 +118,25 @@ from source.
     go test github.com/tensorflow/tensorflow/tensorflow/go
     ```
 
+### Generate Ops Wrapper
+
+Tensorflow ops wrappers in Go is generated by a script. Here is how to generate Go source files for Tensorflow ops.
```

Comment:
Perhaps reword:
Go functions corresponding to TensorFlow operations are generated in `op/wrappers.go`. To regenerate them:

Prerequisites:
- [Protocol buffer compiler (protoc) 3.x](https://github.com/google/protobuf/releases/)
- The TensorFlow repository under GOPATH